hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/TraceabilitySuite.java

```diff
@@ -4475,7 +4474,6 @@ final Stream<DynamicTest> vanillaBytecodeSidecar2() {
     }
 
     @Order(24)
-    @LeakyHapiTest(overrides = {"contracts.sidecars"})
```

Comment:
~~These two tests - 24 and 26 - are no longer "leaky", right?  Because no longer overriding a property?  Maybe they should be moved out of the leaky suite?~~

Oh never mind: These are _also_ leaky because they keep reusing the same transaction name (`TRACEABILITY_TXN`) in test after test.  Which ... they shouldn't do.